lms/envs/common.py

```diff
@@ -2124,6 +2125,13 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
     'openedx.core.djangoapps.site_configuration.middleware.SessionCookieDomainOverrideMiddleware',
 ]
 
+if django.VERSION >= (3, 1):
```

Comment:
[Documenation](https://github.com/jotes/django-cookies-samesite#quickstart) says it should be to the top of middlewares.

Thats why appending seems wrong option instead i check if its greater or equal 3.1 remove it.